examples/plot_05_train_test_split.py

```diff
@@ -0,0 +1,180 @@
+"""
+.. _example_train_test_split:
+
+==========================
+Enhancing train-test split
+==========================
+
+This example illustrates the motivation and the use of skore's
+:func:`skore.train_test_split` to get assistance when developing your
+ML/DS projects.
+"""
+
+# %%
+# Creating and loading the skore project
+# ======================================
+#
+# We start by creating a temporary directory to store our project such that we can
+# easily clean it after executing this example. If you want to keep the project, you
+# have to skip this section.
+
+# %%
+import tempfile
+from pathlib import Path
+
+import skore
+
+temp_dir = tempfile.TemporaryDirectory(prefix="skore_example_")
+temp_dir_path = Path(temp_dir.name)
+
+my_project = skore.create("my_project.skore", working_dir=temp_dir_path)
+
+# %%
+# Train-test split in scikit-learn
+# ================================
+#
+# Scikit-learn holds a function for splitting the data into a training and a testing
+# sets: :func:`sklearn.model_selection.train_test_split`.
+# Its signature is the following:
+#
+# .. code-block:: python
+#
+#     sklearn.model_selection.train_test_split(*arrays, test_size=None, train_size=None, random_state=None, shuffle=True, stratify=None)
```

Comment:
```suggestion
#     sklearn.model_selection.train_test_split(
#         *arrays,
#         test_size=None,
#         train_size=None,
#         random_state=None,
#         shuffle=True,
#         stratify=None
#     )
```
otherwise the text overflows to the right and can't be read